_component_design/selector-strip.md

```diff
@@ -40,20 +106,20 @@ Use a selector strip on forms requiring a user to select from a small group of a
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Best practice
+### Composition
 
-The selector strip component functions best when:
+A single select strip contains the following elements:
 
-- The labels inside the strip are short (1-3 words long max) so the number of options can fit horizontally on a small screen.
-- Checkboxes or radio buttons do not make sense visually.
-- The options are important to the rest of user's workflow and deserve a more prominent visual treatment.
+- **Strip container (required)**: The selector strip container holds the options.
+- **Selected option container (required)**: The selected option container expands to include the option label.
```

Comment:
The selected option container surrounds the selected option label to indicate it has been selected.